internal/zinc-ivy-integration/src/main/scala/sbt/internal/inc/IfMissing.scala

```diff
@@ -0,0 +1,14 @@
+package sbt.internal.inc
+
+sealed trait IfMissing
```

Comment:
We may want to explore to make this stable in the next iteration. That's why I split it up, there was too much logic in its place of origin.